tests/generic_relations_regress/tests.py

```diff
@@ -119,9 +119,9 @@ def test_target_model_len_zero(self):
         note = Note(note='Deserve a bonus', content_object=team1)
         note.save()
 
-    def test_target_model_nonzero_false(self):
+    def test_target_model_bool_false(self):
         """Test related to #13085"""
-        # __nonzero__() returns False -- This actually doesn't currently fail.
+        # __bool__() returns False -- This actually doesn't currently fail.
```

Comment:
If you could give a try at rewriting this docstring to conform to our guidelines about stating the expected behavior, that would be nice.